mentoring/mindset.md

```diff
@@ -0,0 +1,140 @@
+when they are having trouble understanding something.
+It may be more helpful to keep trying different explanations and offering other examples then to simply
+push them with encouragement that can sound to them like "C'mon! Don't you get it yet?"
+
+The solution a mentee arrives at, even after several iterations with a mentor, may not be the most performant,
+or may not be _perfectly_ conformant with coding standards.
+To drive the mentee for a perfect solution is an example of the absolute factual correctness that is not in the spirit of Exercism.
+The solution may be [good enough][good-enough], at which point the refinement of technique can be reserved for another mentoring session.
+
+## The spirit of obligation
+
+A mentor may not be sure of their obligations.
+The obligations are few and simple: to be helpful and encouraging.
+If at any point the mentor feels they can't be either of those, then it may be best to politely suggest that the session end
+so the mentee can resubmit for another mentor.
+It may be best for the mentor to invite the menteee to end the discussion, but if the mentor feels the need to end the discussion,
+it would be polite to do so with a final comment explaining why.
+Although it may be tempting to focus on the faults of the mentee that made the session unsatisfactory,
+it would be more in the spirit of Exercism for the mentor to not focus blame on the mentee. 
+
+## The spirit of cooperation
+
+The cooperative spirit of Exercism is best expressed by not just giving away the (better) solution to the exercise,
+but by describing other appproaches that could be used and leaving it to the mentee to code the implementation of those suggestions.
+The solution could be made _optionally_ available to the mentee in case they get stuck.
+It could be a snippet example in a collapsible details section, or it could be a link to a published solution.
+It would be the mentee's choice whether to expand the section or follow the link.
+
+It's possible the mentee may acknowledge a preference for a suggestion, but say they will keep it in mind for the future.
+The mentor may suggest that another iteration could be submitted that is based on the suggestion,
+but the mentee may decide to end the discussion without modifying their original solution.
+Although it may a bit embarrassing to have a published solution marked as being mentored by you that you think could be better,
+it is in the spirit of cooperation to not push the mentee to implement the suggestion if they seem disinclined.
+
+Another way the cooperative spirit works is by the mentor learning from the mentee.
+During the course of the session the mentee may bring up something the mentor doesn't know.
+A mentor is not expected to present an "all-knowing" facade.
+It's perfectly fine for a mentor to express that they didn't know or had not thought of something the mentee brings up.
```

Comment:
Maybe mention that this will be a great experience for the student too? I loved it when mentors said I'd taught them something.

Reply:
Added a line for that.